src/components/personalia/Personalia.tsx

```diff
@@ -11,9 +11,10 @@ import Link from "next/link";
 
 interface IProps {
   personalia: IPersonalia;
+  mode?: "standard" | "summary";
```

Comment:
Det ser ut som dette bare brukes til å vise/skjule heading? I så fall synes jeg det er mer lesbart med en optional `showHeading?: boolean;` Da slipper man også å sette default verdi på parameteren

Da blir ryddigere i markup også ` {showHeading && <SectionHeading text={personaliaTexts} fallback={textId} />}`

Reply:
@hanneolsen og jeg tenkte på å gjøre den til en boolean, men valget om å gjøre det her slik var såvidt jeg husker for å unngå negativer og dersom personalia skulle vises et annet sed, så er det praktisk om vi ikke trenger å måtte endre koden flere steder. Jeg er åpen til forslag, så hvis dette er et bedre alterntiv, så fikser jeg klart det!

Reply:
Det er litt hips om haps om dette synes jeg. Så lenge vi kun har én endring i de to visningene kan det hende at det er bedre å spesifisere det som en mer eksplisitt prop, som `showHeading`. Men dette er en smakssak, og noe du kan vurdere hva du synes er mest lesbart @beeccy . :) Begge løsningene fungerer bra.